src/test/java/com/midaswebserver/midasweb/services/HashServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package com.midaswebserver.midasweb.services;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.springframework.test.util.AssertionErrors.assertFalse;
+import static org.springframework.test.util.AssertionErrors.assertTrue;
+
+@SpringBootTest
+public class HashServiceImplTest {
```

Comment:
Need more tests for this service, as you're missing a number of obvious tests for this service.